awx/api/templates/instance_install_bundle/group_vars/all.yml

```diff
@@ -7,7 +7,7 @@ receptor_work_commands:
     command: ansible-runner
     params: worker
     allowruntimeparams: true
-    verifysignature: true
+    verifysignature: {{ sign_work }}
```

Comment:
I think this particular instance of this change might have been unintended. This file isn't related to the dev env and I think this will break without a change to ansible/receptor-collection to go along with it.